### DISCUSSION TITLE
Unable to preventDefault inside passive event listener invocation

### DISCUSSION THREAD

**Discussion by franciskisiara** [quasarframework/quasar#17048] [General - Components / Directives / etc]
Upon panning the knob on the q-slider component, the following message gets logged on the console.

`Unable to preventDefault inside passive event listener invocation`

Efforts to resolve the issue have pointed me towards the browser with a common css solution being `style="touch-action: pan-x;"`. However, this doesn't work for me. 

I have tested this on; 
Chrome: issue exists, 
Edge: issue exists, 
Firefox: issue exists 
Safari: issue DOE NOT exist.

**Comment by har1s1m** [comment]
Same here. Did you manage to find solution for this?
